homeassistant/components/zwave_js/api.py

```diff
@@ -2498,3 +2507,86 @@ def _handle_device_added(device: dr.DeviceEntry) -> None:
         )
     ]
     await driver.async_hard_reset()
+
+
+@websocket_api.websocket_command(
+    {
+        vol.Required(TYPE): "zwave_js/node_capabilities",
+        vol.Required(DEVICE_ID): str,
+    }
+)
+@websocket_api.async_response
+@async_handle_failed_command
+@async_get_node
+async def websocket_node_capabilities(
+    hass: HomeAssistant,
+    connection: ActiveConnection,
+    msg: dict[str, Any],
+    node: Node,
+) -> None:
+    """Get node endpoints with their support command classes."""
+
+    def map_command_class(command_class: CommandClassInfoDataType) -> dict:
+        """Map and return only required fields."""
+        copy = dict(command_class.copy())
+        copy["is_secure"] = copy.pop("isSecure")
+        return copy
+
+    connection.send_result(
+        msg[ID],
+        {
+            idx: [
+                map_command_class(command_class)
+                for command_class in endpoint.data["commandClasses"]
```

Comment:
I think we should use the library built api for command classes, ie `endpoint.command_classes` and `CommandClassInfo` and build a dict from scratch, or alternatively add a `to_dict` method to `CommandClassInfo` in the library. We've moved away from using `.data` elsewhere in the integration.

Reply:
Requires https://github.com/home-assistant-libs/zwave-js-server-python/pull/1059